pkg/output/progress.go

```diff
@@ -0,0 +1,98 @@
+package output
+
+import (
+	"time"
+
+	"go.uber.org/atomic"
+
+	"github.com/sirupsen/logrus"
+)
+
+var spinner = []string{"⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷"}
+
+const (
+	timeout      = 10 * time.Second
```

Comment:
When reading the test file I was wondering where does `timeout` come from, as it is exposed in the package maybe we can rename it to `progressTimeout` ?

`timeout => progressTimeout`
`displaySpeed => progressRefreshRate`